.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+<!-- Thanks for adding a resource about remote work to this list! 🎉
+
+Please fill in the below placeholders -->
+
+**[Insert URL you're adding here]**
+
+**[Explain what this URL adds and why it is awesome]**
+
+<!-- Make sure you've read the [Contribution Guidelines](CONTRIBUTING.md) before submitting! Thank you. -->
```

Comment:
IMO it would be fine to ask to actually follow the steps outlined in the CONTRIBUTING.md, rather than merely read the document. In fact, we could even add that PRs that don't follow those guidelines will not be merged, as further motivation. WDYT?

Reply:
Good point. Changed.